app/cookies.js

```diff
@@ -2,6 +2,7 @@ const { cookie: { cookieNameCookiePolicy }, cookiePolicy } = require('./config')
 
 function getCurrentPolicy (request, h) {
   let cookiesPolicy = request.state[cookieNameCookiePolicy]
+  cookiesPolicy = Array.isArray(cookiesPolicy) ? cookiesPolicy[cookiesPolicy.length - 1] : cookiesPolicy
```

Comment:
It would be good to know why, and fix the root cause of why multiple cookies are being created.